src/scene/container/container-mixins/cacheAsTextureMixin.ts

```diff
@@ -92,7 +92,7 @@ export const cacheAsTextureMixin: Partial<Container> = {
     set cacheAsBitmap(val: boolean)
     {
         // #if _DEBUG
-        deprecation('cacheAsBitmap is deprecated, use cacheAsTexture instead.');
+        deprecation('v7.x.x', 'cacheAsBitmap is deprecated, use cacheAsTexture instead.');
```

Comment:
The version here seems wrong to me. We deprecated this in v8.0.0, or maybe more accurately v8.6.0. There are versions consts in deprecation.ts.

Reply:
changed it to 8.6.0 for him